src/Exception/AuthException.php

```diff
@@ -8,14 +8,9 @@ class AuthException extends \Exception
 {
     public function __construct(
         $message = 'There was an error authenticating',
-        $code = 0,
```

Comment:
Since we are going this route, can we switch [formatted exceptions](http://rosstuck.com/formatting-exception-messages/)?